ecosystem/typescript/sdk/src/util.ts

```diff
@@ -7,3 +9,11 @@ export async function sleep(timeMs: number): Promise<null> {
     setTimeout(resolve, timeMs);
   });
 }
+
+export function moveStructTagToParam(moveStructTag: Gen.MoveStructTag): Gen.MoveStructTagParam {
```

Comment:
Where is this used?

Reply:
In a couple of places in the client, e.g. getAccountResource. It accepts a MoveStructTag, because it is more structured (harder to make mistakes) and then converts it into the string, because it needs to be used as a path param.